apps/academy/src/pages/tracks/fundamentals/index.tsx

```diff
@@ -106,7 +106,17 @@ const IntroToEthereumPage = () => {
         trackAuthor="elPiablo, georgemac510"
         trackAuthorDescription="The authors have a wealth of knowledge in the field of education and pedagogy"
         trackAuthorTwitter="@GeorgeMac510, @Skruffster"
-        tags={["Beginner", "Web3", "Eth"]}
+        tags={[
+          "Entry",
+          "Explorer",
+          "Infra",
+          "Blockchain",
+          "Back-end",
+          "Front-end",
+          "Bash",
+          "RPC",
+          "Node",
+        ]}
```

Comment:
max 3?

Reply:
Yes, for the cards. Inside the actual track page, you can add about 10, but not much more, as they need to fit in one 'horizontal line'